### ISSUE TITLE
Errors PDDE checker- Missing address and No inventory records

### ISSUE THREAD

**Issue by Kavitadt** [abtassociates/eva#480]
No Uploads

I am looking for any data issues for ES and SH project types. So I have created a reporting group with all ES and SH project types.

1. I Ran a hashed HMIS csv with my reporting group "ES and SH project types" and uploaded it into eva. The upload is successful and when I go to the "Access Data Quality" tab and click on "Check PDDE's" It shows 2 errors:
Error 1: 
Type: Error, Issue: Missing address, Guidance: Please ensure geography information for projects is complete. There is only one project that comes up with this error and when I go into HMIS to check the project setup the address under the "Provider Profile is complete. Also under the "Standards Information" The CoC code has completed geocode and zip code. So not sure why this comes up as an error for that one project
4. Error2:
Type: Error, Issue: No Inventory records, Guidance: Residential projects should have inventory data. Please enter inventory in HMIS for the project.
This error comes up for all the projects in my reporting group and each one of them has the bed and unit inventory updated

**Expected behavior**
These errors are incorrect and should not be coming up as errors unless I am supposed to look somewhere else for the address and inventory

**Screenshots**
None

**Desktop (please complete the following information):**
 - Operating System: Microsoft Windows 11Pro
 - Browser: Google Chrome

**Smartphone (please complete the following information):**
 - I am not using a phone for this

**Additional context**

**Comment by Kavitadt** [comment]
I think I understood the "no inventory issue". My reporting group has all ES,SH project types but some of those projects were closed before the reporting time frame of 10/01/2022-09/30/2023. Please ignore that issue. Any thoughts on the Missing address error will be appreciated.

**Comment by alyssandrichik** [comment]
@Kavitadt - thanks for reaching out about this! To narrow down what is happening here, we need you to look into a few things for us.

First, can you look at the CSV export you uploaded and confirm the address is there as it should be? It should be in the ProjectCoC file.

Second, can you confirm that you have no impermissible character errors listed in your file structure analysis? This is located on the Upload HMIS CSV Export page. We want to confirm there isn't a weird character in the address that is preventing Eva from reading it correctly. 

Thank you!

**Comment by Kavitadt** [comment]
Hey Alyssa,

Thank you for looking into this, I hope I haven't wasted too much of your time. I apologize, but I figured out why that one project was missing address. The error was correct with Eva. I looked into that project and compared it to another one that was pulling correctly, and noticed that even though this project had the address entered in HMIS it was not set as primary address. I guess the HMIS csv export is looking at projects and picking the address that has "primary address" check marked/selected, even if only one address is in HMIS for that project. It has to be checked as primary address to go through. Thank you once again.
